tasks/trigger/jenkins/0.1/.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: trigger-jenkins-job
+  labels:
+    app.kubernetes.io/version: "0.1"
+spec:
+  description: >-
+    The following task can be used to trigger a Jenkins job using
+    CURL request from a Tekton Task.
+  params:
+    - name: JENKINS_HOST_URL
+      type: string
+      description: Server URL on which Jenkins is running
+    - name: JOB_NAME
+      type: string
+      description: Jenkins Job which needs to be triggered
+    - name: JENKINS_SECRETS
+      type: string
+      description: Jenkins secret containing credentials
+      default: jenkins-credentials
+    - name: JOB_PARAMS
+      type: array
+      description: Extra arguments to append as a part of CURL request
+      default: [""]
+  volumes:
+    - name: jenkins-credentials-volume
+      secret:
+        secretName: jenkins-credentials
+  steps:
+    - name: trigger-pipeline
+      image: registry.access.redhat.com/ubi8/ubi:8.2
+      volumeMounts:
+        - name: konflux-test-infra-volume
```

Comment:
jenkins-credentials-volume

Reply:
maybe that `volumeMount` is not needed at all? I see there are [secrets loaded in the env section already](https://github.com/konflux-ci/tekton-integration-catalog/pull/72/files#diff-402c5c867734253dc437be52558c523239d33bd7a65be2a15c5f15c2da905d7aR103)